### PR TITLE
Stats info page

### PR DESCRIPTION
**Pull request by caitmuenster**
Creates a new page for [Monitoring Extension Usage Statistics](https://docs.google.com/document/d/1N5fKmVbVm-ejfI5JLjoxKrQDT3MCZy5ScnvW_c9phcU/edit?ts=5f208493#heading=h.k7p9jijqku9g).

@muffinresearch, I ran into the following problems when checking out on local -- can you offer guidance? 

* New image (src/assets/img/documentation/manage/manage_my_submissions_statistics.png) doesn't show 
* Sidebar doesn't populate even after adding site name and id to `src/data/pages.json`

### CODE REVIEW

File: src/data/pages.json [hunk 1]
```diff
@@ -1095,6 +1095,10 @@
             "title": "Stay informed when Firefox changes",
             "id": "stay-informed-when-firefox-changes"
           },
+          {
```
[the remark this answers is not in the document]
Comment (muffinresearch) [reply src/data/pages.json:1098]: FWIW this is exactly the stuff that I'd hope we can eventually remove with https://github.com/mozilla/extension-workshop/issues/659 and https://github.com/mozilla/extension-workshop/issues/673

[the remark this answers is not in the document]
Comment (caitmuenster) [reply src/data/pages.json]: Thanks @muffinresearch. I ran into trouble with this -- I tried copy/pasting your patch into the file and got this error: 

Watching "src/assets/js/*.js" ..
> Having trouble parsing data file ./src/data/pages.json

`TemplateDataParseError` was thrown
> Unexpected token + in JSON at position 41391

`SyntaxError` was thrown:
    SyntaxError: Unexpected token + in JSON at position 41391
        at parse (<anonymous>)
        at TemplateData._parseDataFile (/Users/cneiman/Git/extension-workshop/node_modules/@11ty/eleventy/src/TemplateData.js:351:16)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "build:serve" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Can you help troubleshoot? Sorry to be a pain.

[the remark this answers is not in the document]
Comment (muffinresearch) [reply src/data/pages.json]: Sure, I've just pushed a commit that should fix it.

**Comment by caitmuenster** [comment]
Thank you! 🙏🏼

**Pull request merged**